test/unit/entities/DutchQuote.test.ts

```diff
@@ -292,6 +297,22 @@ describe('DutchQuote', () => {
       // should have extra adjustment for gas to amountOut
       expect(result.amountOutEnd.lte(slippageAdjustedAmountOut)).toBeTruthy();
     });
+
+    it('Succeeds - Gets portion from classic', () => {
```

Comment:
shall we add a test for rfq too?

Reply:
all these testa are just for classic right, even though we are on the DutchQuote file